src/scss/misc/_colors.scss

```diff
@@ -18,7 +18,7 @@ $colors: (
   assets: (
     oil: (color: #729e00, icon: 'fa-database'),
     gas: (color: #ff7a28, icon: 'fa-fire'),
-    recycle: (color: #f1b71c, icon: 'fa-recycle'),
+    recycle: (color: #b11f24, icon: 'fa-recycle'),
```

Comment:
We don't know what the final color/shape solution will look like until we test the different options with our users and that won't be for a little while. We might use shapes to differentiate assets and let colors signify something else. Coy has decided red will be our recycle color until we hear differently from our users when that time comes.